experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -474,22 +476,36 @@ export class FetchInstrumentation extends InstrumentationBase<
     };
   }
 
-  private _applyAttributesAfterFetch(
-    span: api.Span,
-    request: Request | RequestInit,
-    result: Response | Error
-  ) {
-    const applyCustomAttributesOnSpan =
-      this._getConfig().applyCustomAttributesOnSpan;
-    if (applyCustomAttributesOnSpan) {
+  private _applyAttributesBeforeFetch(span: api.Span, request: Request | RequestInit) {
+    const applyCustomAttributesOnReq =
+      this._getConfig().applyCustomAttributesOnReq;
+    if (applyCustomAttributesOnReq) {
```

Comment:
Try to use early exists, like:

```
if (!applyCustomAttributesOnReq) return
```

And then afterwards you don't have another indent.